app/src/main/res/layout/widget_entry.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Is this file used? It has (nearly) the same content as `widget_single_note_content.xml` which is used by the widget. Maybe, this file can be removed.